nemo_curator/modules/filter.py

```diff
@@ -11,13 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import pandas as pd
+from dask import dataframe as dd
```

Comment:
If you are only using `make_array_nonempty`, you can also make the import a bit more targeted:

```python
from dask.dataframe.extensions import make_array_nonempty
```